contents/tutorials/stripe-reports.md

```diff
@@ -0,0 +1,185 @@
+---
+title: How to set up Stripe reports
+date: 2024-06-21
+author:
+ - ian-vanagas
+tags:
+ - data warehouse
+---
+
+For your startup to become a real business, it needs to make money. Stripe is the most popular choice for helping you do this. Creating and analyzing reports for your Stripe data helps you understand how you are making money and how you can improve.
+
+This tutorial shows you how to sync your Stripe data to PostHog and then create a report of insights like customer count, gross revenue, recurring revenue, revenue churn, and more.
```

Comment:
Cutting out the first two sentences since it feels a bit like ELI5ing what stripe is
```suggestion
Creating and analyzing reports for your Stripe data helps you understand how you are making money and how you can improve. This tutorial shows you how to sync your Stripe data to PostHog and then create a report of insights like customer count, gross revenue, recurring revenue, revenue churn, and more.
```